test/common/utils.cpp

```diff
@@ -395,8 +395,8 @@ TEST_CASE("random_choice isn't obviously wrong") {
 }
 
 TEST_CASE("randomly_capitalize isn't obviously wrong") {
-    auto lower_string = "abcdefghij";
-    auto upper_string = "ABCDEFGHIJ";
+    auto lower_string = "abcdefghijklmnopqrstuvz";
+    auto upper_string = "ABCDEFGHIJKLMNOPQRSTUVZ";
```

Comment:
This is another, unrelated small fix that I committed on this PR to unbreak the build. Strictly speaking it can go into another PR, but if we don't squash it can as well go in with this PR.